server/services/store/sqlstore/category_boards.go

```diff
@@ -78,39 +82,50 @@ func (s *SQLStore) addUserCategoryBoard(db sq.BaseRunner, userID, categoryID, bo
 			"create_at",
 			"update_at",
 			"delete_at",
-		).
-		Values(
-			utils.NewID(utils.IDTypeNone),
-			userID,
-			categoryID,
-			boardID,
-			utils.GetMillis(),
-			utils.GetMillis(),
-			0,
-		).Exec()
+			"sort_order",
+		)
 
-	if err != nil {
+	now := utils.GetMillis()
+	for boardID, categoryID := range boardCategoryMapping {
+		query = query.
+			Values(
+				utils.NewID(utils.IDTypeNone),
+				userID,
+				categoryID,
+				boardID,
+				now,
+				now,
+				0,
+				0,
+			)
+	}
+
+	if _, err := query.Exec(); err != nil {
 		s.logger.Error("addUserCategoryBoard error", mlog.Err(err))
 		return err
 	}
 	return nil
 }
 
-func (s *SQLStore) deleteUserCategoryBoard(db sq.BaseRunner, userID, boardID string) error {
+func (s *SQLStore) deleteUserCategoryBoard(db sq.BaseRunner, userID string, boardIDs []string) error {
```

Comment:
Probably this should be rename to `deleteUserCategoryBoards` with the `s` at the end.

Reply:
Done.